.github/workflows/release.yml

```diff
@@ -40,6 +40,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
         with:
+          ref: ${{ inputs.ref }}
```

Comment:
Fail fast if the ref is not correct. Otherwise, it's required to wait for the Buildkite pipeline to run